src/components/ClockFace.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from 'react';
+import classnames from 'classnames';
+import SettingsModal from './SettingsModal';
+
+const ClockFace = () => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [isActive, setIsActive] = useState(false);
+  const [currentSessionIndex, setCurrentSessionIndex] = useState(0);
+  // in seconds
+  const [sessions, setSessions] = useState([60, 5, 10, 10]);
+  const [currentSeconds, setCurrentSeconds] = useState(sessions[currentSessionIndex]);
```

Comment:
It's really confusing to use `currentSessionIndex` here since you could have just use 0. If you want to make it really robust you can create a constant variable `INIT_SESSION_INDEX = 0` and apply to both places.
```suggestion
  const [currentSeconds, setCurrentSeconds] = useState(sessions[0]);
```